app/src/main/res/values/colors.xml

```diff
@@ -3,6 +3,7 @@
     <color name="colorPrimary">#536dfe</color>
     <color name="colorPrimaryDark">#0043ca</color>
     <color name="colorAccent">#ffd740</color>
+    <color name="colorBlue">#0000ff</color>
```

Comment:
This _cannot_ fit into the theme. Use colorPrimary instead if you want a blue.

Reply:
I've made the change and pushed it